### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by pjeweb**
## Describe your changes

Explain generating secrets better and add python option. Also improve various other things in the README

## Notes for testing your change

Read the README and try the new suggested commands.

### CODE REVIEW

**Comment by MattIPv4** [comment]
Instead of doing `"Generate secrets"`, we could do `[Generate secrets](#generate-secrets)` to deep-link. Or, you could get really fancy and do `[Generate secrets]` and then `[Generate secrets]: #generate-secrets` at the end of the file I think?

File: README.md [hunk 1]
```diff
@@ -71,19 +71,30 @@ Hey there! Welcome to Alveus.gg! There's a few ways that you can help contribute
...
+- Learn more about the stack at [Create T3 App - Introduction](https://create.t3.gg/en/introduction)
+- You can use the Prisma Studio to view your database. Launch it with `pnpm prisma studio`
+
+### Generate secrets
+
+We use Base64-encoded random strings for various secrets. To generate these secrets you can use OpenSSL or Python. OpenSSL should be preinstalled on most Unix-like systems (Linux, macOS, WSL).  If neither is installed on your system, you may need to install it yourself.
+
+- Using OpenSSL (Linux, macOS, WSL):
+  - Generate a 32-byte secret: `openssl rand -base64 32`
+  - Generate a 24-byte secret: `openssl rand -base64 24`
```
Comment (pjeweb) [inline README.md:94]: No need to specify Linux, macOS, WSL here, you can install openssl.exe on windows as well.

File: README.md [hunk 2]
```diff
@@ -71,19 +71,30 @@ Hey there! Welcome to Alveus.gg! There's a few ways that you can help contribute
...
    2. Fill in the S3 section with your S3-compatible storage info
-   3. The vapid keys for web notifications have to be generated using `npx web-push generate-vapid-keys`
-   4. Next Auth secret (`NEXTAUTH_SECRET`), Twitch EventSub API secret (`TWITCH_EVENTSUB_SECRET`) and Action API secret (`ACTION_API_SECRET`) have to generated using `openssl rand -base64 32`
-   5. The `DATA_ENCRYPTION_PASSPHRASE` has to generated using `openssl rand -base64 24`
-   6. You may define privileged users once they have signed in via the `SUPER_USER_IDS` variable (using comma separated values)
-5. Push the database schema to the new database using `npx prisma db push` from within `apps/website`.
+   3. The vapid keys for web notifications have to be generated using `pnpx web-push generate-vapid-keys`
+   4. The Next Auth secret (`NEXTAUTH_SECRET`), Twitch EventSub API secret (`TWITCH_EVENTSUB_SECRET`) and Action API secret (`ACTION_API_SECRET`) have to be filled with 32-byte Base64-encoded secrets. See "Generate secrets" below.
+   5. The Data encryption passphase (`DATA_ENCRYPTION_PASSPHRASE`) has to be filled with a 24-byte Base64-encoded secret. See "Generate secrets" below.
```
Comment (pjeweb) [inline README.md:79]:
```suggestion
   5. The data encryption passphrase (`DATA_ENCRYPTION_PASSPHRASE`) has to be filled with a 24-byte Base64-encoded secret. See "Generate secrets" below.
```

**Comment by MattIPv4** [comment]
@pjeweb do you want me to take this branch over and get it landed?

**Comment by pjeweb** [comment]
> @pjeweb do you want me to take this branch over and get it landed?

yes please. sorry i forgot about this completely :)

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Comment by MattIPv4** [comment]
@pjeweb I'll let you merge this if my commit looks good, so it is "reviewed" by you given it was my push

**Pull request merged**